src/xenia/gpu/d3d12/premake5.lua

```diff
@@ -39,6 +39,7 @@ project("xenia-gpu-d3d12-trace-viewer")
     "xenia-ui",
     "xenia-ui-d3d12",
     "xenia-vfs",
+    "xenia-patcher",
```

Comment:
For easier maintenance, these lists should be sorted alphabetically unless the order matters (if it does, a comment explaining that is needed).